src/PrestaShopBundle/Resources/config/services/adapter/employee.yml

```diff
@@ -21,3 +21,50 @@ services:
     class: 'PrestaShop\PrestaShop\Adapter\Profile\Employee\CommandHandler\BulkDeleteEmployeeHandler'
     tags:
       - { name: tactician.handler, command: PrestaShop\PrestaShop\Core\Domain\Profile\Employee\Command\BulkDeleteEmployeeCommand }
+
+  prestashop.adapter.employee.query_handler.get_employee_for_editing:
+    class: 'PrestaShop\PrestaShop\Adapter\Profile\Employee\QueryHandler\GetEmployeeForEditingHandler'
+    tags:
+      - { name: tactician.handler, command: PrestaShop\PrestaShop\Core\Domain\Profile\Employee\Query\GetEmployeeForEditing }
+
+  prestashop.adapter.employee.form_access_checker:
+    class: 'PrestaShop\PrestaShop\Adapter\Employee\EmployeeFormAccessChecker'
+    arguments:
+      - '@prestashop.adapter.data_provider.employee'
+      - '@prestashop.core.admin.tab.repository'
+
+  prestashop.adapter.employee.command_handler.add_employee_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\Profile\Employee\CommandHandler\AddEmployeeHandler'
+    arguments:
+      - '@prestashop.core.crypto.hashing'
+      - '@prestashop.core.employee.profile_access_checker'
+    tags:
+      - { name: tactician.handler, command: PrestaShop\PrestaShop\Core\Domain\Profile\Employee\Command\AddEmployeeCommand }
+
+  prestashop.adapter.employee.command_handler.edit_employee_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\Profile\Employee\CommandHandler\EditEmployeeHandler'
+    arguments:
+      - '@prestashop.core.crypto.hashing'
+      - '@prestashop.core.employee.profile_access_checker'
+      - '@prestashop.adapter.data_provider.employee'
+      - '@prestashop.adapter.legacy.context'
+    tags:
+      - { name: tactician.handler, command: PrestaShop\PrestaShop\Core\Domain\Profile\Employee\Command\EditEmployeeCommand }
+
+  prestashop.adapter.employee.default_avatar_provider:
+    class: 'PrestaShop\PrestaShop\Adapter\Employee\DefaultAvatarProvider'
+    arguments:
+      - '@prestashop.adapter.tools'
+
+  prestashop.adapter.employee.data_provider:
+    class: 'PrestaShop\PrestaShop\Adapter\Employee\EmployeeDataProvider'
+
+  prestashop.adapter.employee.navigation_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\Employee\NavigationHandler'
+    arguments:
+      - '@prestashop.adapter.legacy.context'
+
+  prestashop.adapter.employee.form_language_changer:
+    class: 'PrestaShop\PrestaShop\Adapter\Employee\FormLanguageChanger'
+    arguments:
+      - '@prestashop.adapter.legacy.context'
```

Comment:
Please finish the file with a blank line

Reply:
there is a blank line actually, but github is not showing it. If you click "view file" it will show that this file contains 71 lines, while here it shows only 70.

Reply:
Very confusing from github 😅